### PR TITLE
Feat: User Welcome and Main Page

### PR DESCRIPTION
**Pull request by Husain01**
-> Add User Onboarding 
-> Add Greetings
-> Add Time
-> Add Main Focus Input
-> Add Background Image fetching from API
-> Add Quote fetching from API

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,28 +1,25 @@
-.App{
-/* z-index: -1; */
-height: 100vh;
-background-repeat: no-repeat;
+.App {
+  /* z-index: -1; */
```
Comment (simran61) [inline src/App.css:2]: can remove comment if not in use

**Review by simran61 — APPROVED** [review]
Looks good
Nice folder structure
Clean UI

**Review by sri-rishi — APPROVED** [review]
code and ui looks good, you can merge

**Review by manojncsoni007 — APPROVED** [review]
- clean code
- good naming conventions
good to merge

**Pull request merged**